### ISSUE TITLE
setBounds not working on macos using Window class

### ISSUE THREAD

**Issue by KASOGIT** [sentialx/node-window-manager#23]
My use-case: i have an integer representing the id of the window i wanna share then i use directly `new Window(id)` and getInfo give me exactly the right window but setBounds not working with this method.

My work around: loop over getWindows and use the window matching the id.

Expected behavior: first method should work directly.

Some code illustrating what i'm saying:

```
    // first not working method
    const selectedWin = new Window(parsedId);

    // second working workaround
    const selectedWin = windowManager.getWindows().find(window => {
      return window.id === parsedId;
    });

    selectedWin.setBounds({
      x: screenBounds.x + sideBarWidth + 15,
      y: screenBounds.y,
      width: screenBounds.width - sideBarWidth - 15,
      height: screenBounds.height,
    });
```

Best regards

**Comment by sentialx** [comment]
@KASOGIT macOS or Windows?

**Comment by sentialx** [comment]
Fixed in 57821c3b8f84a8c641ee86e6e7f96e5bb44f1e83

**Comment by KASOGIT** [comment]
> Fixed in [57821c3](https://github.com/sentialx/node-window-manager/commit/57821c3b8f84a8c641ee86e6e7f96e5bb44f1e83)

Thanks <3
